core/src/banking_stage/transaction_scheduler/scheduler_controller.rs

```diff
@@ -272,7 +272,7 @@ impl SchedulerController {
 
         let (received_packet_results, receive_time_us) = measure_us!(self
             .packet_receiver
-            .receive_packets(recv_timeout, remaining_queue_capacity));
+            .receive_packets(recv_timeout, remaining_queue_capacity, |_| true));
         saturating_add_assign!(self.timing_metrics.receive_time_us, receive_time_us);
```

Comment:
there was a merge conflict here due to metric accumulation change in master. see 1fc684bad8e0e236b41ca38fd8fac4e0ce9f5d3f for resolve